src/storybook-components/_data/demoData.js

```diff
@@ -1416,12 +1416,14 @@ const demoData = {
         textArea: null,
         dateRange: ['', ''],
         date: '',
-        address: {},
+        address: {
+          state: 'vic'
+        },
         checkbox: true,
         select: '',
         multiselect: null,
-        checklistlistbox: null,
-        checklistdropdown: null,
+        checklistlistbox: ['topic_a'],
```

Comment:
It's good to test how a form element responds when it has a default value. but there's also benefit testing an item when it isn't set with a value. I'm not sure how best to provide all potential states - would including these fields as null alongside these prepopulated ones be better? It's a pass, but maybe a broader discussion for the future.

Reply:
Hi @alan-cole We will add Vue test utils soon https://vue-test-utils.vuejs.org/ for better component unit testing. Demo data will mostly be used for visual demos.

Reply:
@alan-cole in this case we need the default value to show functionality that was missing (selects weren't initialising with default value).